src/main/java/dev/walshy/sfmetrics/charts/ServerSizeChart.java

```diff
@@ -2,38 +2,53 @@
 
 import org.bstats.bukkit.Metrics.SimplePie;
 import org.bukkit.Bukkit;
-
+import org.bukkit.Server;
+
+/**
+ * This {@link SimplePie} allows us to see how big a {@link Server} is (player-wise).
+ * 
+ * @author TheBusyBiscuit
+ *
+ */
 public class ServerSizeChart extends SimplePie {
 
     public ServerSizeChart() {
         super("server_size", () -> {
             int players = Bukkit.getOnlinePlayers().size();
 
-            if (players < 10) {
-                return "0-10";
+            if (players < 5) {
+                return "0-5";
+            }
+
+            if (players < 15) {
+                return "5-15";
             }
 
             if (players < 25) {
-                return "10-25";
+                return "15-25";
             }
 
             if (players < 50) {
                 return "25-50";
             }
 
+            if (players < 75) {
+                return "50-75";
+            }
+
             if (players < 100) {
-                return "50-100";
+                return "75-100";
             }
 
-            if (players < 200) {
-                return "100-200";
+            if (players < 150) {
+                return "100-150";
             }
 
-            if (players < 300) {
-                return "200-300";
+            if (players < 200) {
+                return "150-200";
             }
```

Comment:
These should really all be else-ifs

(Also man, these below 15 players are going to be majority)

Reply:
That's why I split it up further in adding a "0-5" slice.

Reply:
Should we even send if no one is on?